app/views/CanWeContactByPhoneView.scala.html

```diff
@@ -48,9 +48,10 @@
     pageTitle = Some(messages("tai.service.navTitle")),
     authedUser = incomingAuthedUser,
     backLinkContent = Some(Messages("tai.back-link.upper")),
-    backLinkUrl = Some(viewModel.backUrl)
-) {
+    backLinkUrl = Some(viewModel.backUrl),
+    formForErrorSummary = Some( yesNoForm)
```

Comment:
might as well remove this it has already been done in other PR's, will save the merge conflicts

Reply:
removed